packages/admin/src/NotificationManager.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Filament;
+
+use Livewire\Response;
+
+class NotificationManager
+{
+    protected static $notifications = [];
+
+    public static function notify(string $status, string $message): void
+    {
+        session()->push('notifications', [
+            'id' => uniqid(),
+            'status' => $status,
+            'message' => $message,
+        ]);
+
+        self::$notifications = session()->get('notifications');
+    }
+
+    public static function handleLivewireResponses($component, Response $response): Response
```

Comment:
Can we typehint the `$component` prop to be a `Livewire\Component` object?

Reply:
I think so. Not sure why I haven't done it.